php-frontend/src/main/java/org/sonar/php/metrics/CognitiveComplexityVisitor.java

```diff
@@ -242,26 +243,44 @@ public void visitGotoStatement(GotoStatementTree tree) {
     super.visitGotoStatement(tree);
   }
 
+  public void handleCognitiveComplexityPIPE(List<SyntaxToken> flatOperators) {
```

Comment:
Nitpick naming suggestion
```suggestion
  public void handleCognitiveComplexityPipe(List<SyntaxToken> flatOperators) {
```